packages/camlp5/camlp5.8.00/opam

```diff
@@ -31,6 +31,7 @@ depends: [
   "ocaml" {>= "4.02" & < "4.13.0"}
   "conf-perl"
   "conf-perl-packages"
```

Comment:
This (and similar in several places) needs to change now that the packages have been renamed

Reply:
Yep, I noticed and pushed just before your comment came in.